README.md

```diff
@@ -118,42 +105,31 @@ window.metrilo.identify('johnybravo@gmail.com') // This would be called whenever
 
 When they submit the order, you'll need to send the order event from your backend:
 
-```
-curl -X POST https://trk.mtrl.me/order" -i -H "Content-Type: text/plain" -d \
-'{
-  "time": 1518004715732,
-  "token": "sh0p-t0k3n",
-  "params": {
-    "id": "314",
-    "createdAt": "1518004715732",
-    "email": "johnybravo@gmail.com",
-    "amount": "10.42",
-    "status": "completed",
-    "products": [
-      {
-        "productId": "2",
-        "quantity": 1
-      }
-    ]
-  }
-}'
+```bash
+$ echo -n '{"time": 1518004715732, "token": "sh0p-t0k3n", "params": {"id": "314", "createdAt": "1518004715732", "email": "johnybravo@gmail.com", "amount": "10.42", "status": "completed", "products": [{"productId": "2", "quantity": 1 } ] } }' | openssl dgst sha256 -hmac 'm3trilo-secr3t'
+4401f8a26f2d2bf86df5354a8f7213464dd3da8368cf381af5494d518ba291c1
+
+$ curl -X POST https://trk.mtrl.me/v2/order" -i -H "Content-Type: text/plain" -H "X-Digest: 4401f8a26f2d2bf86df5354a8f7213464dd3da8368cf381af5494d518ba291c1" -d '{"time": 1518004715732, "token": "sh0p-t0k3n", "params": {"id": "314", "createdAt": "1518004715732", "email": "johnybravo@gmail.com", "amount": "10.42", "status": "completed", "products": [{"productId": "2", "quantity": 1 } ] } }'
+
 ```
 
 And that's it! Now when all events are processed by Metrilo, they will be attributed to the `johnybravo@gmail.com` user. You can check that out in the `Customer Database` tab in your [Metrilo project](https://app.metrilo.com).
 
 :information_source: Please allow for up to several minutes for all events to be processed.
 
-:information_source: If all backend calls return 200 and you're missing events - double check your input data.
+:information_source: If all backend  calls return 200 and you're missing events - double check your input data.
 
 ## API reference
 
-Our API documentation is written according to the [OpenAPI Specification](https://swagger.io/docs/specification/about/). You can find all of the endpoints the Metrilo API serves and what parameters they require [here](https://app.swaggerhub.com/apis/metrilo/api/1.0.0).
+Our API documentation is written according to the [OpenAPI Specification](https://swagger.io/docs/specification/about/). You can find all of the endpoints the Metrilo API serves and what parameters they require [here](https://app.swaggerhub.com/apis/metrilo/api/2.0.0).
 
-:information_source: The base url for each request is `https://trk.mtrl.me`. Don't forget that you need to add your Metrilo `API token` as a `token` parameter in the request body.
+:information_source: The base url for each request is `https://trk.mtrl.me/v2`. Don't forget that you need to add your Metrilo `API token` as a `token` parameter in the request body.
```

Comment:
Another option you have is to push tags for the corresponding versions instead of duplicating the whole thing - so that you could just switch what version you're viewing in Github. A lot of gems are written this way

Reply:
@evgeni-spasov-metrilo what do you think about this one?

Reply:
In that case, what will happen if I have to update the API v1 docs?

Reply:
We'll have to branch-off v1 if we need to security fix it. Let the versions live their own live in separate files for now. If they get too many, we can always move to tags.

Reply:
Yeah, that's a fair point. Let it be